src/Cake.Common/Tools/ILRepack/ILRepackSettings.cs

```diff
@@ -0,0 +1,186 @@
+
+        /// <summary>
+        /// Gets or sets a file to enable logging to (no logging if null or empty)
+        /// </summary>
+        /// <value>The log file.</value>
+        public string Log { get; set; }
+
+        /// <summary>
+        /// Gets or sets the target assembly version.
+        /// </summary>
+        /// <value>The version.</value>
+        public System.Version Version { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to merge types with identical names into one
+        /// </summary>
+        /// <value><c>true</c> if types with identical names should be merged into one; otherwise, <c>false</c>.</value>
+        public bool Union { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to disable symbol file generation
+        /// </summary>
+        /// <value><c>true</c> if debug symbols should be disabled; otherwise, <c>false</c>.</value>
+        public bool NDebug { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to copy assembly attributes (by default only the primary assembly attributes are copied)
+        /// </summary>
+        /// <value><c>true</c> if assembly attributes should be copied; otherwise, <c>false</c>.</value>
+        public bool CopyAttrs { get; set; }
+
+        /// <summary>
+        /// Gets or sets the assembly file to take attributes from
+        /// </summary>
+        /// <value>The assembly file to take attributes from.</value>
+        public string Attr { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to allow multiple attributes (if type allows)
+        /// </summary>
+        /// <value><c>true</c> if multiple attributes should be allowed; otherwise, <c>false</c>.</value>
+        public bool AllowMultiple { get; set; }
+
+        /// <summary>
+        /// Gets or sets the specify target assembly kind (library, exe, winexe supported, default is same as first assembly)
+        /// </summary>
+        /// <value>The kind of the target assembly to create.</value>
+        public ILMerge.TargetKind TargetKind { get; set; }
+
+        /// <summary>
+        /// Gets or sets the target platform (v1, v1.1, v2, v4 supported)
+        /// </summary>
+        /// <value>The target platform.</value>
+        public ILMerge.TargetPlatformVersion? TargetPlatform { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to merge XML documentation as well
+        /// </summary>
+        /// <value><c>true</c> if xml documents should be merged; otherwise, <c>false</c>.</value>
+        public bool XmlDocs { get; set; }
+
+        /// <summary>
+        /// Gets or sets the paths to search directories for referenced assemblies (can be specified multiple times)
+        /// </summary>
+        /// <value>The libs.</value>
+        public List<string> Libs { get; set; }
```

Comment:
Yeah I was debating that but wasn't sure if it wanted just the filename or the full path of the assembly.... Never used this option.